sell-loot.lic

```diff
@@ -19,24 +19,45 @@ class SellLoot
 
     settings = get_settings
     town_data = get_data('town')
-    @actual_fucking_hometown = settings.hometown
+    @character_hometown = settings.hometown
     @hometown = town_data[settings.hometown]
     @bankbot_name = settings.bankbot_name
     @bankbot_deposit_threshold = settings.bankbot_deposit_threshold
     @bankbot_room_id = settings.bankbot_room_id
+    bankbot_enabled = settings.bankbot_enabled
+    @local_currency = town_data[@character_hometown]['currency']
 
     sell_gems("#{settings.gem_pouch_adjective} pouch") if settings.sell_loot_pouch
-
     check_spare_pouch(settings.spare_gem_pouch_container, settings.gem_pouch_adjective) if settings.spare_gem_pouch_container
 
     sell_bundle if settings.sell_loot_bundle
 
-    return if settings.sell_loot_skip_bank
-    exchange_coins
+    return if settings.sell_loot_skip_bank && !bankbot_enabled
+
+    @keep_coppers = parse(settings.sell_loot_money_on_hand.split(' '))
+    @keep_coppers = @bankbot_deposit_threshold if bankbot_enabled
```

Comment:
If `bankbot_enabled` is true, should we instead keep the max of `settings.sell_loot_money_on_hand` and `@bankbot_deposit_threshold`?

Reply:
If not, maybe we should have an entry in `validate.lic` that says you have `sell_loot_money_on_hand` set, but also `bankbot_enabled` is true. Something to indicate that `sell_loot_money_on_hand` is ignored.

Reply:
Ideally I'd like to have one setting.  How would you feel above removing bankbot_deposit_threshhold?

Reply:
So... after further review on this.  Both settings are needed because bank withdraw uses "1 silver" and bankbot uses "tip 100".  They don't mix very well.  (Or at all)

I recommend they stay independent and people can just figure it out.  One goes for bank withdrawals, one is for bank deposit.

Bankbot deposit threshhold only works for the local currency based on current hometown settings. (assuming you are deposting all currencies.)  I considered added the option to keep multiple currencies but decided it would be not be used.